frontend/src/components/_common/atoms/Slider/style.ts

```diff
@@ -0,0 +1,78 @@
+import styled from '@emotion/styled';
+
+const SliderContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: 2rem;
+`;
+
+const SliderTrack = styled.div`
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+
+  width: 100%;
+  height: 0.6rem;
+  border-radius: 0.3rem;
+
+  background-color: ${({ theme }) => theme.baseColors.grayscale[200]};
+`;
+
+const SliderRange = styled.div<{ left: number; right: number }>`
+  position: absolute;
+  top: 50%;
+  left: ${({ left }) => `${left}%`};
+  right: ${({ right }) => `${right}%`};
+  transform: translateY(-50%);
+
+  height: 0.6rem;
+  border-radius: 0.3rem;
+  background-color: ${({ theme }) => theme.baseColors.purplescale[200]};
+`;
+
+const SliderThumb = styled.input`
+  position: absolute;
+  top: 50%;
+  width: 100%;
+  height: 0;
+  background: transparent;
+  pointer-events: none;
+
+  -webkit-appearance: none;
+  appearance: none;
+
+  &::-moz-range-thumb {
+    width: 1.6rem;
```

Comment:
Firefox에서 시험해보면 아래 이미지와 같이 SliderThumb 요소가 찌그러집니다.

<img width="316" alt="image" src="https://github.com/user-attachments/assets/46c8c846-0ebf-4390-b2bb-27bc5b7a9b60">

요 부분에 한해서 width를 1rem으로 맞춰주면 정상적으로 출력되네요 :)

<img width="311" alt="image" src="https://github.com/user-attachments/assets/48a6c06b-b090-489f-acf8-a144a6902a33">

```suggestion
    width: 1rem;
```

Reply:
와 ㅋㅋ 환경별로 코드가 달라진다는게 진짜 이상하네용..